packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -264,57 +278,17 @@ contract OPStackManager is ISemver, Initializable {
         upgradeAndCall(output.opChainProxyAdmin, address(output.optimismMintableERC20FactoryProxy), impl.logic, data);
 
         impl = getLatestImplementation("L1CrossDomainMessenger");
-        // TODO add this check back in
-        // require(
-        //     impl.logic == referenceAddressManager.getAddress("OVM_L1CrossDomainMessenger"),
-        //     "OpStackManager: L1CrossDomainMessenger implementation mismatch"
-        // );
         data = encodeL1CrossDomainMessengerInitializer(impl.initializer, output);
         upgradeAndCall(output.opChainProxyAdmin, address(output.l1CrossDomainMessengerProxy), impl.logic, data);
 
         impl = getLatestImplementation("L1StandardBridge");
         data = encodeL1StandardBridgeInitializer(impl.initializer, output);
         upgradeAndCall(output.opChainProxyAdmin, address(output.l1StandardBridgeProxy), impl.logic, data);
 
-        // -------- TODO: Placeholders --------
-        // For contracts we don't yet deploy, we set the outputs to  dummy proxies so they have code to pass assertions.
-        output.disputeGameFactoryProxy = DisputeGameFactory(deployProxy(l2ChainId, output.opChainProxyAdmin, "1"));
-        output.disputeGameFactoryImpl = DisputeGameFactory(deployProxy(l2ChainId, output.opChainProxyAdmin, "2"));
-        output.anchorStateRegistryProxy = AnchorStateRegistry(deployProxy(l2ChainId, output.opChainProxyAdmin, "3"));
-        output.anchorStateRegistryImpl = AnchorStateRegistry(deployProxy(l2ChainId, output.opChainProxyAdmin, "4"));
-        output.faultDisputeGame = FaultDisputeGame(deployProxy(l2ChainId, output.opChainProxyAdmin, "5"));
-        output.permissionedDisputeGame = PermissionedDisputeGame(deployProxy(l2ChainId, output.opChainProxyAdmin, "6"));
-        output.delayedWETHPermissionedGameProxy =
-            DelayedWETH(payable(deployProxy(l2ChainId, output.opChainProxyAdmin, "7")));
-        output.delayedWETHPermissionlessGameProxy =
-            DelayedWETH(payable(deployProxy(l2ChainId, output.opChainProxyAdmin, "8")));
-
         // -------- Finalize Deployment --------
         // Transfer ownership of the ProxyAdmin from this contract to the specified owner.
         output.opChainProxyAdmin.transferOwnership(_input.roles.opChainProxyAdminOwner);
 
-        // Correctness checks.
-        // TODO these currently fail in tests because the tests use dummy implementation addresses that have no code.
-        // if (output.systemConfigProxy.owner() != _input.roles.systemConfigOwner) {
```

Comment:
I'm guessing we can kick this can down the road and this will be checked as part of: https://github.com/ethereum-optimism/optimism/issues/11814

Reply:
Some of these are now checked in `assertValidDeploy`  in `DeployOPChain`, we are basically just going to port all the `ChainAssertions.sol` checks into that contract